install-vela-cli.sh

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env bash
```

Comment:
Why we need this script?

Reply:
Oops, didnt intend for this file to be added. Will fix it